app/templates/reports/manage_report.html

```diff
@@ -0,0 +1,106 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% set deletion_endpoint = 'reports.delete_report_request' %}
+
+{% set endpoints = [
+    ('reports.report_info', 'Report information'),
+    ('reports.edit_report_info', 'Edit report'),
+    (deletion_endpoint, 'Delete report')
+] %}
+
+{% macro navigation(items) %}
+    <div class="ui vertical fluid secondary menu">
+        {% for route, name in items %}
+            {% set href = url_for(route, report_id=report.id) %}
+            <a class="item {% if request.endpoint == route %}active{% endif %}" href="{{ href }}">
+                {{ name }}
+            </a>
+        {% endfor %}
+    </div>
+{% endmacro %}
+
+{% macro report_info(report) %}
+    <table class="ui compact definition table">
+        <tr><td>Report ID</td><td>{{ report.id }}</td></tr>
+        <tr><td>Vehicle ID</td><td>{{ report.vehicle_id }}</td></tr>
+        <tr><td>License Plate</td><td>{{ report.license_plate }}</td></tr>
+        <tr><td>Bus Number</td><td>
+                {{ report.bus_number if report.bus_number else '' }}
+            </td></tr>
+        <tr><td>LED Screen Number</td><td>
+                {{ report.led_screen_number if report.led_screen_number else '' }}
+            </td></tr>
+        <tr><td>Location</td><td>{{ report.location }}</td></tr>
+        <tr><td>Date</td><td>{{ report.date.strftime('%m-%d-%Y') }}</td></tr>
+        <tr><td>Time</td><td>{{ report.date.strftime('%I:%M %p') }}</td></tr>
+        <tr><td>Duration (h:m:s)</td><td>{{ report.duration}}</td></tr>
+        <tr><td>Agency</td><td>{{ report.agency.name }}</td></tr>
+        <tr><td>Picture</td><td><img src="{{ report.picture_url }}" alt="report picture" width="200"</td></tr>
+        <!-- TODO add width attr to styles -->
+        <tr><td>Description</td><td>{{ report.description }}</td></tr>
+    </table>
+{% endmacro %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <a class="ui basic compact button" href="{{ url_for('reports.view_reports') }}"> <!-- TODO my reports -->
```

Comment:
Make sure to go through all remaining TODOs you added and decide which need to be in this pr